### DISCUSSION TITLE
sidekiq -> DragonflyDb integration

### DISCUSSION THREAD

**Discussion by romange** [sidekiq/sidekiq#5717] [Q&A]
hello.
I would like to validate whether sidekiq can work with DragonflyDB instead of Redis. Are there any (regression) tests that I can run using already existing "redis" endpoint ?

**Comment by mperham** [comment]
If you install it at localhost:6379, you can run the Sidekiq test suite against it.

1. Check out the code
2. `bundle`
3. `bundle exec rake`
